### PR TITLE
Add SetCookie and GetCookie method for Context

### PR DESCRIPTION
**Pull request by se77en**

### CODE REVIEW

File: context.go [hunk 1]
```diff
@@ -321,6 +322,64 @@ func (c *Context) Header(key, value string) {
...
+		}
+	}
+
+	c.Writer.Header().Add("Set-Cookie", cookie.String())
+}
+
+func (c *Context) GetCookie(name string) string {
+	cookie, err := c.Request.Cookie(name)
+	if err != nil {
+		return ""
```
Comment (ChristopherRabotin) [inline context.go]: Shouldn't this log the error instead of dying silently?

File: context.go [hunk 2]
```diff
@@ -321,6 +322,64 @@ func (c *Context) Header(key, value string) {
 	}
 }
 
+func (c *Context) SetCookie(name string, value string, options ...interface{}) {
+	cookie := http.Cookie{}
+	cookie.Name = name
+	cookie.Value = url.QueryEscape(value)
+
+	if len(options) > 0 {
```
Comment (ChristopherRabotin) [inline context.go]: This sounds risky. Why just add the max age parameter in the SetCookie signature?

File: context.go (hunk 1 above)
Comment (se77en) [inline context.go]: Getting cookie mustn't be a empty string, so I think logging an error is unnecessary

File: context.go (hunk 2 above)
Comment (se77en) [inline context.go]: fix it

**Comment by javierprovecho** [comment]
@se77en thank you, merged in develop (4892650ef8c095468cf3a1d8f166c28d45205653)

**Pull request closed without merging**